.env.kesaseteli-backend.example

```
@@ -13,6 +13,7 @@ VTJ_PASSWORD=
 VTJ_TIMEOUT=30
 EXCEL_DOWNLOAD_BATCH_SIZE=50
 EXCLUDE_2026_EXCEL_FIELDS=True
+UPDATE_COMPANY_FROM_YTJ_ON_SUBMIT=True
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The UPDATE_COMPANY_FROM_YTJ_ON_SUBMIT key should go before the VTJ_PASSWORD key

(UnorderedKey)